Lib/multiprocessing/forkserver.py

```diff
@@ -144,19 +160,31 @@ def ensure_running(self):
                 # all client processes own the write end of the "alive" pipe;
                 # when they all terminate the read end becomes ready.
                 alive_r, alive_w = os.pipe()
+                # A short lived pipe to initialize the forkserver authkey.
+                authkey_r, authkey_w = os.pipe()
                 try:
-                    fds_to_pass = [listener.fileno(), alive_r]
+                    fds_to_pass = [listener.fileno(), alive_r, authkey_r]
+                    main_kws['authkey_r'] = authkey_r
                     cmd %= (listener.fileno(), alive_r, self._preload_modules,
-                            data)
+                            main_kws)
```

Comment:
Whoa, it's never occurred to me to use in-place modulo for string formatting.  TIL 😄

Reply:
Yeah...  this is very old code... I'd like to rewrite this to be more clear but i'm saving that for a separate refactoring-only rainy day PR.  =)